Packs/ML/Scripts/HashIncidentsFields/HashIncidentsFields.yml

```diff
@@ -0,0 +1,125 @@
+args:
+- default: false
+  description: Additional text by which to query incidents.
+  isArray: false
+  name: query
+  required: false
+  secret: false
+- default: false
+  description: A comma-separatetd list of incident types by which to filter.
+  isArray: false
+  name: incidentTypes
+  required: false
+  secret: false
+- default: false
+  description: 'The start date by which to filter incidents. Date format will be the
+    same as in the incidents query page, for example: "3 days ago", ""2019-01-01T00:00:00
+    +0200").'
+  isArray: false
+  name: fromDate
+  required: false
+  secret: false
+- default: false
+  description: 'The end date by which to filter incidents. Date format will be the
+    same as in the incidents query page, for example: "3 days ago", ""2019-01-01T00:00:00
+    +0200").'
+  isArray: false
+  name: toDate
+  required: false
+  secret: false
+- default: false
+  defaultValue: '3000'
+  description: The maximum number of incidents to fetch. The default value is 3000.
+  isArray: false
+  name: limit
+  required: false
+  secret: false
+- auto: PREDEFINED
+  default: false
+  defaultValue: 'false'
+  description: Whether to query and fetch incident context. Can be "true" or "false".
+    The default is "false".
+  isArray: false
+  name: includeContext
```

Comment:
you don't need this argument, because you query the context separately, remove it